src/Components/SkillsSection.tsx

```diff
@@ -0,0 +1,76 @@
+import { CheckIcon, FlagIcon } from "@heroicons/react/outline";
+
+const features = [
+  {
+    name: "Programming Languages & Engines",
+    description: "JavaScript(ES6) / HTML5 / CSS3 / Node",
+  },
+  {
+    name: "Front-End Frameworks",
+    description: "React (with Hooks & Redux) / Vue (with Vuex)",
+  },
+  {
+    name: "Front-End Libraries",
+    description: "Material UI / Semantic UI / Vuetify / Bootstrap / Tailwind",
+  },
+  {
+    name: "Back-End Frameworks",
+    description: "Express / Mongoose",
+  },
+  {
+    name: "Database Types",
+    description: "MongoDB",
+  },
+  {
+    name: "Programming Tools",
+    description:
+      "Visual Studio Code / CLI / Heroku / Git / GitHub / Postman / npm & yarn",
+  },
+  {
+    name: "Community Tools",
+    description: "Slack / Zoom / Stack Overflow / Medium / JIRA",
+  },
+  {
+    name: "Currently Learning:",
+    description: "Golang / AWS / SQL / TypeScript",
+  },
+];
+
+export default function SkillsSection() {
+  return (
+    <div className="bg-white">
+      <div className="max-w-7xl mx-auto py-16 px-4 sm:px-6 lg:py-8 lg:px-8">
+        <div className="max-w-3xl mx-auto text-center">
+          <h2 className="text-3xl font-extrabold text-gray-900">
+            Technical Skills
+          </h2>
+        </div>
+        <dl className="mt-12 space-y-10 sm:space-y-0 sm:grid sm:grid-cols-2 sm:gap-x-6 sm:gap-y-12 lg:grid-cols-4 lg:gap-x-8">
+          {features.map((feature) => (
+            <div key={feature.name} className="relative">
+              <dt>
+                {feature.name === "Currently Learning:" ? (
```

Comment:
The problem with conditions like this is if you want to change the title, you have to change the condition as well (and it's something one can easily forget about).
The objects in the array `features` could have another property like `"status": "done"/"in-progress"` for this purpose and then the name is independent of the status